examples/v1beta1/pipelineruns/alpha/pipeline-object-param.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+  name: mock-clone-task
+spec:
+  params:
```

Comment:
Looking at `examples/v1beta1/pipelineruns/alpha/pipeline-emitting-results.yaml` together, I realize another example is missing is that: a task is using a previous task's results in the params as a whole. Example with object keys: https://github.com/tektoncd/pipeline/blob/main/examples/v1beta1/pipelineruns/task_results_example.yaml and https://github.com/tektoncd/pipeline/blob/main/docs/pipelines.md#passing-one-tasks-results-into-the-parameters-or-when-expressions-of-another

Reply:
This file is the example for object param only. Object result (i.e. using a previous task's object result) example is in the file [/pipeline-object-results.yaml](https://github.com/tektoncd/pipeline/blob/main/examples/v1beta1/pipelineruns/alpha/pipeline-object-results.yaml).